### PR TITLE
Fix typos in norne restart deck

### PR DESCRIPTION
**Pull request by joakim-hove**
Small typos in deck - has this ever worked?

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: norne/INCLUDE/BC0407_RESTART.SCH [hunk 1]
```diff
@@ -3148,9 +3148,9 @@ WCONHIST
     'E-3AH'      'OPEN'      'RESV'   1609.046    178.782 626599.537  5* /
      'E-2H'      'OPEN'      'RESV'   5792.532      0.000 949276.709  5* /
     'E-4AH'      'OPEN'      'RESV'   1485.125      0.000 141254.548  5* /
+/
 
-
-WCONINJ
+WCONINJE
```
Comment (bska) [inline norne/INCLUDE/BC0407_RESTART.SCH:3153]: Please check this closely as that change could change the semantics of the input deck.  `WCONINJ` is the less flexible precursor to `WCONINJE`.

Comment (joakim-hove) [reply norne/INCLUDE/BC0407_RESTART.SCH:3153]: Yes but: 

1. We do not support `WCONINJ` at all - the json keyword is just a placeholder.
2. All other ocurences in the deck are `WCONINJE`.

So - I am very confident this is a typo.

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
Build check is green.  I'll merge into master.

**Pull request merged**